packages/@react-aria/overlays/test/usePopover.test.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {fireEvent, render} from '@react-spectrum/test-utils-internal';
+import React, {useRef} from 'react';
+import {useOverlayTriggerState} from '@react-stately/overlays';
+import {usePopover} from '../';
+
+function Example() {
+  const triggerRef = useRef(null);
+  const popoverRef = useRef(null);
+  const state = useOverlayTriggerState({isOpen: true});
+  const {popoverProps} = usePopover({triggerRef, popoverRef}, state);
+
+  return (
+    <div>
+      <div ref={triggerRef} />
+      <div {...popoverProps} ref={popoverRef} />
+    </div>
+  );
+}
+
+describe('usePopover', () => {
+  it('should not add scroll listener', () => {
```

Comment:
Thanks so much for the PR.

This test is very much about the implementation details. Could we make it more about how a user would interact with it and come across this in real life? That way if we change how this is implemented down the line, we won't break the test and we'll know we haven't broken the intended behavior.